pkgs/development/libraries/science/math/nccl/tests.nix

```diff
@@ -0,0 +1,54 @@
+{ lib
+, stdenv
```

Comment:
```suggestion
```

`stdenv` isn't used.

Unrelated, but if you haven't checked out https://github.com/oxalica/nil, I highly recommend it! It flags unused terms and even has type inference!